tests/testsuite/rustdoc.rs

```diff
@@ -61,7 +61,11 @@ fn rustdoc_foo_with_bar_dependency() {
     foo.cargo("rustdoc -v -- --cfg=foo")
         .with_stderr(
             "\
+[DOCUMENTING] bar v0.0.1 ([..])
```

Comment:
This test broke somehow, its now documenting bar when it didn't used to and I don't understand why. I got it to pass by adjusting the output to expect it to document bar, but I doubt this is correct. Also this will fail sometimes because the Checking and Documenting lines for bar can swap places.